src/hotspot/cpu/s390/gc/shared/barrierSetNMethod_s390.cpp

```diff
@@ -26,26 +26,26 @@
 #include "code/codeBlob.hpp"
 #include "code/nativeInst.hpp"
 #include "code/nmethod.hpp"
+#include "gc/shared/barrierSetAssembler.hpp"
 #include "gc/shared/barrierSetNMethod.hpp"
 #include "utilities/debug.hpp"
 
 class NativeMethodBarrier: public NativeInstruction {
   private:
-    static const int PATCHABLE_INSTRUCTION_OFFSET = 3*6; // bytes
 
     address get_barrier_start_address() const {
       return NativeInstruction::addr_at(0);
     }
 
     address get_patchable_data_address() const {
-      address inst_addr = get_barrier_start_address() + PATCHABLE_INSTRUCTION_OFFSET;
+      address inst_addr = get_barrier_start_address() + BarrierSetAssembler::PATCHABLE_SEQ_START_OFFSET;
 
       DEBUG_ONLY(Assembler::is_z_cfi(*((long*)inst_addr)));
       return inst_addr + 2;
```

Comment:
Maybe use `get_barrier_start_address() + BarrierSetAssembler:: PATCHABLE_BARRIER_VALUE_OFFSET`? `inst_addr` could be removed.